src/js/containers/bulkDownload/BulkDownloadPageContainer.jsx

```diff
@@ -127,6 +127,16 @@ export class BulkDownloadPageContainer extends React.Component {
             }
         }
 
+        if (formState.agencyType) {
+            delete params.filters.agency;
+            const topTierAgencyParams = { type: formState.agencyType === 'awarding_agency' ? 'awarding' : 'funding', tier: "toptier", name: formState.agency.name };
+            params.filters.agencies = [topTierAgencyParams];
+            if (formState.subAgency.name && formState.subAgency.name !== 'Select a Sub-Agency') {
```

Comment:
After looking through backend code more there is one more update for the logic to build the "agencies" filter. There are two cases:

1) User provides only a toptier agency which will produce:
```
{
   "name": <TOPTIER_AGENCY_NAME>,
   "tier": "toptier",
   "type": <SELECTED_AGENCY_TYPE>,
}
```

2) User provides a toptier and subtier agency which will produce:
```
{
   "name": <SUBTIER_AGENCY_NAME>,
   "tier": "subtier",
   "type": <SELECTED_AGENCY_TYPE>,
   "toptier_name": <TOPTIER_AGENCY_NAME>
}
```

`toptier_name` needs to be added to the API Contract but it is supported.